backend/djangoindia/bg_tasks/send_update.py

```diff
@@ -12,20 +15,36 @@ def send_mass_update_email_task(update_id):
         update = Update.objects.get(id=update_id)
     except Update.DoesNotExist:
         raise ValueError("Update not found")
+    
+    logger.info(f"Starting email task for update ID {update_id} with {update.recipients.count()} recipients.")
+
+    failed_recipients = []
 
-    email_objs = []
     for subscriber in update.recipients.all():
-        email_tuple = (
-            f"Django India: {update.get_formatted_type()}",
-            update.html_template,
-            settings.DEFAULT_FROM_EMAIL,
-            [subscriber.email],
-        )
-        email_objs.append(email_tuple)
-    try:
-        send_mass_mail((email for email in email_objs), fail_silently=False)
-        update.mail_sent = True
-        update.save()
-    except Exception as e:
-        # Didn't throw error at django level even for wrong emails.
-        print(f"Failed to send email : {str(e)}")
+        try:
+            plain_text_content = strip_tags(update.html_template)
+
+            email = EmailMultiAlternatives(
+                subject=f"Django India: {update.get_formatted_type()}",
```

Comment:
I am not sure if this approach is right or we should be moving ahead with `Update` model itself.

I suggest a better approach here at  https://github.com/djangoindia/djangoindia.org/issues/230#issuecomment-2491682713